### PR TITLE
[READY]Adds the Smooth-Headed (super bald) quirk

### PR DESCRIPTION
**Pull request by Mickyan**
You lost the genetic lottery and are pretty embarassed about it! 
You get a small mood boost from wearing a wig (you get one at roundstart that matches your hair preference) and get sad from having your head left uncovered

## Changelog
:cl:Mickyan, Qustinnus
add: Added the Smooth-Headed quirk. Make sure nobody finds out that you're very, very bald.
/:cl:

### CODE REVIEW

File: code/datums/traits/neutral.dm [hunk 1]
```diff
@@ -168,3 +168,54 @@
...
+		W.hairstyle = old_hair
+	W.update_icon()
+	var/list/slots = list (
+		"head" = ITEM_SLOT_HEAD,
+		"backpack" = ITEM_SLOT_BACKPACK,
+		"hands" = ITEM_SLOT_HANDS,
+	)
+	H.equip_in_one_of_slots(W, slots , qdel_on_fail = TRUE)
+
+/datum/quirk/bald/proc/check_headgear(mob/user, obj/item/hat, slot)
```
Comment (Qustinnus) [inline code/datums/traits/neutral.dm]: can you add a comment and mention that slot is always null if its called by the dropped signal? Might make it a bit more clear to others.

**Comment by trollbreeder** [comment]
What if you choose an alien race that doesn't have hair?

File: code/datums/traits/neutral.dm [hunk 2]
```diff
@@ -168,3 +168,54 @@
...
+	var/mob/living/carbon/human/H = quirk_holder
+	if (slot == ITEM_SLOT_HEAD)
+		var/obj/item/I = H.head
```
Comment (Qustinnus) [inline code/datums/traits/neutral.dm]: you already have this var in the params of the proc

**Comment by Iamgoofball** [comment]
BALD.

**Comment by wesoda25** [comment]
powerful

File: code/datums/traits/neutral.dm (hunk 1 above)
Comment (ExcessiveUseOfCobblestone) [reply code/datums/traits/neutral.dm]: doc req

**Comment by Mickyan** [comment]
Props to Qustinnus for rewriting most of this in a sane way

**Comment by necromanceranne** [comment]
What if you're proud of your bald head?

**Comment by MMMiracles** [comment]
proud baldies are in denial

**Comment by ATH1909** [comment]
> What if you choose an alien race that doesn't have hair?

You can still wear a wig, then.

Maybe add a special sprite for B A L D moths and plasmamen, though?

**Comment by ishitbyabullet** [comment]
abductor nerf
bald people nerf
rob robinson nerf

**Comment by Mickyan** [comment]
> abductor nerf
> bald people nerf
> rob robinson nerf

confirmed super bald

**Comment by gitbirb** [comment]
You gotta be accepting the bald plasmemes too, get em' a plasmeme wig or something

**Pull request merged**